build.gradle

```diff
@@ -0,0 +1,233 @@
+
+import net.ltgt.gradle.errorprone.CheckSeverity
+
+plugins {
+  id 'com.diffplug.gradle.spotless' version '3.16.0'
```

Comment:
We probably want to adopt https://github.com/pinterest/ktlint instead.

Reply:
switched to ktlint.  still needs to be configured